### DISCUSSION TITLE
通过springboot+mybatis访问doris，能感知到doris集群吗？

### DISCUSSION THREAD

**Discussion by yclxiao** [apache/doris#14009] [A - General / Q&A]
通过springboot+mybatis访问doris，能感知到doris集群吗？

**Comment by zy-kkk** [comment]
和连接mysql的方式一样，把端口换成doris的mysql server 端口就可以(默认9030)

**Reply by yclxiao** [reply]
如果数据分布在doris的5个BE节点上，springboot+mybatis的访问方式，能把5个节点的数据都查到吧？

**Reply by zy-kkk** [reply]
可以的，查询是通过连接FE来查询，你只需要通过springboot+mybatis连接集群中的一个FE节点即可，而无需关心数据具体是在哪个BE

**Reply by yclxiao** [reply]
好的，感谢 @zy-kkk
